### DISCUSSION TITLE
Possibility of debugging using STM32CubeIde

### DISCUSSION THREAD

**Discussion by Piki1989** [OpenI6X/opentx#257] [Q&A]
Hello,

Is there any way to configure flysky fs-i6x hardware with st-link debugger (directly connected) to have a way to real time debugging using STM32CubeIDE ?
For example adding additional switch for DFU a then triggering debug mode ?

Best Regards,
Michal Pikus

**Comment by ajjjjjjjj** [comment]
Hi, there is a way to debug using gdb:
https://fishpepper.de/2016/09/17/openground-part-4-running-gdb-on-stm32f0/
